### PR TITLE
Check for null before sending to a specific connection id

### PR DESCRIPTION
**Pull request by davidfowl**
- Added some tests for the DefaultHubLifetimeManager

#905

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Add similar checks for `RedisHubLifetimeManager`?

**Comment by davidfowl** [comment]
Which checks? The argument checks?

**Comment by BrennanConroy** [comment]
Yeah, null checks

**Comment by davidfowl** [comment]
Yea I originally didn't want to add them since I'd have to add them in every provider but that's fine.

File: src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs [hunk 1]
```diff
@@ -302,6 +312,16 @@ public override Task OnDisconnectedAsync(HubConnectionContext connection)
 
         private async Task<bool> AddGroupAsyncCore(string connectionId, string groupName)
         {
+            if (connectionId == null)
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs]: For consistency, move this to `AddGroupAsync`

**Pull request merged**